docs/next.config.mjs

```diff
@@ -50,6 +48,37 @@ const rootPackage = loadPackageJson();
 const nextConfig = {
```

Comment:
It'd be ideal if we could follow the accepted way of adding items to the default config like -

```js
export default withDocsInfraPipeline(nextConfig);
```

Reply:
Good idea, should we replace `withDocsInfra()`?

Reply:
I think the current one will still be there. We could have a new one, like `withDocsInfraDemo()` or similar since it mainly affects the demos.

Reply:
What would you think about `withDocsInfra(nextConfig, { demos: true })`